### ISSUE TITLE
Missing function AttachAudioStreamProcessor and DetachAudioStreamProcessor.

### ISSUE THREAD

**Issue by Hultan** [gen2brain/raylib-go#425]
I can't complete the examples/audio/stream_effects, the last example that is missing in the audio section at [raylib.com](https://www.raylib.com/examples.html), because the functions AttachAudioStreamProcessor and DetachAudioStreamProcessor are missing. If you could add them, I could (probably) finish off the examples/audio section.

**Comment by JupiterRider** [comment]
Right. They are missing in the CGO Version. They have to be added in raylib/raudio.go

**Comment by JupiterRider** [comment]
We also should review DetachAudioStreamProcessor and DetachAudioMixedProcessor in the purego versions, which cant work, because it always detaches a new callback.

**Comment by JupiterRider** [comment]
@gen2brain What do you think about using `purego.NewCallback` in the cgo version as well?
With that, the code would be much cleaner, because we don't need wrapper functions anymore.

Otherwise the code will be much more muddled: https://github.com/gen2brain/raylib-go/commit/c2c8150de9c2eb880a9f04c9c0021b110279a76a

**Comment by gen2brain** [comment]
@JupiterRider If possible that would be nice. I remember I had an issue though, there is a limit of callbacks one can have if you fire NewCallback every time, I had to do something like this https://github.com/gen2brain/webp/blob/main/webp_dynamic.go#L220 and https://github.com/gen2brain/webp/blob/main/webp_dynamic.go#L278. 

Though, I don't like breaking support for 32-bit platforms. Luckily there are not many callbacks in raylib.

**Comment by JupiterRider** [comment]
@gen2brain Do you think `SetAudioStreamCallback` even works? (added by you in #279)
I ask because you can only set one global callback and not one callback per AudioStream.

**Comment by gen2brain** [comment]
@JupiterRider I don't think I ever tested it.

**Comment by Rockenstien** [comment]
How can I use AttachAudioStreamProcessor ?

**Comment by JupiterRider** [comment]
@Rockenstien for now manually (by writing your functions in C) or you use the purego version.

@gen2brain do you have any Ideas, how to solve this issue?

**Comment by gen2brain** [comment]
@JupiterRider Sorry, I have no idea and currently have no time to work on that.

**Comment by gen2brain** [comment]
@JupiterRider It is probably something like one C-exported Go function that would always be called; that function can call all Go-added functions to some map.
